README.md

```diff
@@ -34,7 +34,7 @@ For an existing react application, follow the steps below:
 1. Load the fiori-fundamentals styles. If using create-react-app, this will be in `App.css`.
 
     ```scss
-    @import '~fiori-fundamentals/dist/fiori-fundamentals.min.css';
+    @import '~node_modules/fiori-fundamentals/dist/fiori-fundamentals-ie11.min.css';
```

Comment:
I thought the tilde (`~`) character already resolved to the project's `node_modules` directory.

Reply:
https://stackoverflow.com/questions/39535760/what-does-a-tilde-in-a-css-url-do Looks like you are right. Updating.